dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorHelper.java

```diff
@@ -129,14 +134,20 @@ private boolean isIndexHealthy(final String index, final long timeOut) throws Th
                 .withFallback(Boolean.FALSE)
                 .get(this.failFastBooleanPolicy(timeOut, () -> {
                     try{
-                        final Client client=new ESClient().getClient();
-                        return client.prepareSearch(index)
-                                .setQuery(QueryBuilders.termQuery("_type", "content"))
-                                .setSize(0)
-                                .execute()
-                                .actionGet(INDEX_OPERATIONS_TIMEOUT_IN_MS)
-                                .getHits()
-                                .getTotalHits() > 0;
+
+                        SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+                        searchSourceBuilder.query(QueryBuilders.matchAllQuery());
+                        searchSourceBuilder.size(0);
+                        searchSourceBuilder.timeout(TimeValue
+                                .timeValueMillis(INDEX_OPERATIONS_TIMEOUT_IN_MS));
+                        searchSourceBuilder.fetchSource(new String[] {"inode"}, null);
+                        SearchRequest searchRequest = new SearchRequest();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'searchRequest' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)